fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -576,6 +576,30 @@ rocksdb::ColumnFamilyOptions getCFOptions() {
 	return options;
 }
 
+rocksdb::ColumnFamilyOptions getCFOptionsForInactiveShard() {
+	auto options = getCFOptions();
+	// never slowdown ingest.
+	options.level0_file_num_compaction_trigger = (1 << 30);
+	options.level0_slowdown_writes_trigger = (1 << 30);
+	options.level0_stop_writes_trigger = (1 << 30);
+	options.soft_pending_compaction_bytes_limit = 0;
```

Comment:
Just curios, why we are not setting pending_compaction_bytes_limit to big numbers?
I think 0 should speed up compaction.

Reply:
Auto compaction is disabled for this CF. Technically the number doesn't matter.